pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <!-- We use the PackageVersion property defined in Packaging.props for the Prerelease packages which are using it to be built --> 
     <_PreReleasePackageVersion>$(PackageVersion)</_PreReleasePackageVersion> 
-    <PackageVersion>3.1.0</PackageVersion>
-    <ServiceModelVersion>4.8.1</ServiceModelVersion>
+    <PackageVersion>3.1.1</PackageVersion>
+    <ServiceModelVersion>4.8.2</ServiceModelVersion>
```

Comment:
You need to update System.Data.SqlClient version as well in L33

Reply:
I just checked, and last stable package for Service model projects is 4.7.0. should this be changed to 4.7.0 ?

Reply:
Yeah, we should match: https://github.com/dotnet/corefx/blob/c7235990fa6d24ee0e21c8c25a56f2db68cda548/pkg/Microsoft.Windows.Compatibility/externalIndex.json